Procfile

```diff
@@ -0,0 +1 @@
+web: python simple_app/app_simple.py -p $PORT -i '0.0.0.0' -f 'SIMPLE-binary/SIMPLE.db'
```

Comment:
Can just give the file "SIMPLE.db" as it is already symlinked down to the binary?
Doesn't make a difference though